### PR TITLE
Please update the following.

### PR DESCRIPTION
**Pull request by Mridulbirla13**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- A GCI- 2017 task
- Added the profile information in the Students.yml section.
- Added the image in the img/student section.
-The screenshot proof of my task is 
screenshot

### CODE REVIEW

**Review by ParthS007 — CHANGES_REQUESTED** [review]
Please use Image of only `240 * 240 ` dimension

**Review by isuruAb — COMMENTED** [review]
please resize your image to 240x240

**Review by the-codingninja — CHANGES_REQUESTED** [review]
Please resize your image to fix travis

**Comment by abishekvashok** [comment]
Please resolve the conflicts.

File: _data/students.yml [hunk 1]
```diff
@@ -163,3 +163,10 @@
   country: India
   github: lavavi
   image: lavavi.jpg
+  
+- name: Mridul Birla
+  country: India
+  github: Mridulbirla13
+  image: Mridul.jpg
+  twitter: @mridulbirla13
```
Comment (akshatnitd) [inline _data/students.yml:171]: Please remove the ```@```, as the travis build fails because of this.

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Make requested changes and resolve conflicts

**Comment by ParthS007** [comment]
closing this as a duplicate of #326

**Pull request closed without merging**